### PR TITLE
Add configuration option to enable/disable essays

### PR DESCRIPTION
**Pull request by Kaladin13**
We should be able to configure this behavior from config without changing the source code every time

Lagus Maksim, P33081

### CODE REVIEW

**Comment by ryukzak** [comment]
Fully agree with you, but we need to do it in a little bit different way:

1. We need to enable/disable essays by one.
2. Current way to make configuration is too complicated and magic. So it was reworked here: https://github.com/ryukzak/course-bot/pull/36 (will merge when I fix production config)
3. The simplest solution -- comment out essay from the config, if we don't need it right now.
4. Better solution -- do it like for quiz. Put all essays config in `{:essays {1 {...} 2 {...}}}`. Generate command for each mention essay.

**Review by ryukzak — CHANGES_REQUESTED** [review]
- conflict
- separate configuration for different essays

**Comment by Kaladin13** [comment]
@ryukzak 
I've added essays enabling based on their presence in new config, so when we add configuration for next essay, it will be  automatically added to the bot

File: src/course_bot/csa.clj [hunk 1]
```diff
@@ -69,23 +69,29 @@
...
-;      (essay/assignreviewers-talk db conf "essay3")
-;      (essay/review-talk db conf "essay3")
-;      (essay/myfeedback-talk db conf "essay3")
+      (if (contains? conf :essay3)
+       (do
+         (essay/submit-talk db conf "essay3")
+         (essay/status-talk db conf "essay3")
+         (essay/assignreviewers-talk db conf "essay3")
+         (essay/review-talk db conf "essay3")
+         (essay/myfeedback-talk db conf "essay3")))
```
Comment (ryukzak) [inline src/course_bot/csa.clj:94]: - When is much better.
- Do you actually check it? I don't sure hat `nil` value is available in `handlers/defhandler`?

**Comment by ryukzak** [comment]
Implemented in https://github.com/ryukzak/course-bot/pull/52 (sorry, not clean)

Thank you for the attempt!

**Pull request closed without merging**